streaming/receiver.c

```diff
@@ -719,10 +722,12 @@ static int rrdpush_receive(struct receiver_state *rpt)
     rrdcontext_host_child_disconnected(rpt->host);
 
 #ifdef ENABLE_ACLK
-    // in case we have cloud connection we inform cloud
-    // new child connected
-    if (netdata_cloud_setting)
-        aclk_host_state_update(rpt->host, 0);
+    // Child disconnection
+    if (netdata_cloud_setting) {
+        struct aclk_database_worker_config *wc = (struct aclk_database_worker_config *)rpt->host->dbsync_worker;
```

Comment:
Why do you cast the type explicitly? I don't think you need it. Moreover, you didn't do that a few lines before.

Reply:
Will update